ydb/library/yql/providers/dq/opt/logical_optimize.cpp

```diff
@@ -336,7 +310,13 @@ class TDqsLogicalOptProposalTransformer : public TOptimizeTransformerBase {
         auto hopSetting = GetSetting(aggregate.Settings().Ref(), "hopping");
         if (input) {
             if (hopSetting) {
-                return RewriteAsHoppingWindow(node, ctx, input.Cast());
+                bool analyticsHopping = Config->AnalyticsHopping.Get().GetOrElse(false);
+                const auto lateArrivalDelay = TDuration::MilliSeconds(Config->WatermarksLateArrivalDelayMs
+                    .Get()
+                    .GetOrElse(TDqSettings::TDefault::WatermarksLateArrivalDelayMs));
+                bool defaultWatermarksMode = Config->WatermarksMode.Get() == "default";
+                bool asyncActor = Config->ComputeActorType.Get() != "async";
```

Comment:
Don't understand that. Maybe asyncActor = Config->ComputeActorType.Get() == "async"; ?

Reply:
Fixed